tests/integration/cmd_debug_test.go

```diff
@@ -34,10 +34,10 @@ var _ = Describe("odo debug command tests", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
-	Context("odo debug on a nodejs component", func() {
+	Context("odo debug on a nodejs:8 component", func() {
```

Comment:
why was this changed to `8`?

Reply:
Because the latest nodejs image is not working with debug mode and hence also it's now shown as fully supported

Reply:
We need to resolve the debug mode issue for latest node in a separate PR

Reply:
Issue for the broken debug mode https://github.com/openshift/odo/issues/2359#issuecomment-550284845

Reply:
We also have an issue which mentions that we should run test across all the image versions. So we have recorded everything that has to be done after this fix